wallet/bindings/java/lib/src/test/java/org/iota/WalletTests.java

```diff
@@ -49,4 +53,32 @@ public void testGetAccounts() throws WalletException {
             System.out.println(x);
     }
 
+    @Test
+    public void testGenerateAddress() throws WalletException {
+        AddressGenerationOptions.GenerateAddressOptions options = new AddressGenerationOptions.GenerateAddressOptions().withLedgerNanoPrompt(false);
+        AddressGenerationOptions addressOptions = new AddressGenerationOptions().withOptions(options);
+
+        String address = wallet.generateAddress(0, false, 0, options, "rms");
+        assertEquals("rms1qpx0mcrqq7t6up73n4na0zgsuuy4p0767ut0qq67ngctj7pg4tm2ynsuynp", address);
+
+        // generated account at line 77 has first 2 pre-made, so we check against the third
```

Comment:
line 77 isn't right (anymore?)

Reply:
youre right, ill change to it just say the account address later